src/getCoinCombination.test.js

```diff
@@ -7,4 +7,63 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
```

Comment:
I can't see any changes in your PR, revise it and fix according to the comment below

Reply:
Ok, good changes, but revise please test description and fix it please, check comments below